plugins/__init__.py

```diff
@@ -5,7 +5,7 @@
 # Copyright:    (c) Sychev Pavel 2017
 # Licence:      GPL
 
-import base
+from .base import *
```

Comment:
nit: import * considered bad - https://stackoverflow.com/questions/2386714/why-is-import-bad